var/spack/repos/builtin/packages/decentralized-internet/package.py

```diff
@@ -0,0 +1,19 @@
+##############################################################################
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class DecentralizedInternet(Package):
```

Comment:
```suggestion
class DecentralizedInternet(MakefilePackage):
```
This tells Spack that your package is just another makefile package, so you can remove the `install` method below, as it is the default for this base class.